projects/js-upload-api/src/Privacy.ts

```diff
@@ -144,7 +145,7 @@ export class Privacy {
      * @throws Error if the upload fails
      * 
      */
-    public async upload(client: Client): Promise<Privacy> {
+    public async upload(client: Client | ClientPkey): Promise<Privacy> {
```

Comment:
thought the generated docs/types would read better if these types were specifically `Client | ClientPkey` rather than the base class but i can definitely change it. definitely cleaner to do it this way since its an internal class as mentioned below.

Reply:
That's reasonable, In which case we should export the closed union type instead

Reply:
https://github.com/graphistry/graphistry-js/pull/156/commits/90391c5f77112e180ecd1dea50c19491e89c1815